### PR TITLE
Health portal refactor rebased

### PR DESCRIPTION
**Pull request by binocarlos**
Add the health portal templates to px-deploy

There are two new templates:

 * `health-portal` = single cluster with 3 scenarios
 * `health-portal-dr` = two clusters with 4 scenarios
 
The `assets/health-portal` folder has the deployment.yaml and script to manage the installation of the health-portal app onto the first cluster stood up by px-deploy

It works by generating a base64 encoded JSON packet that has the:

 * apiServer
 * token
 * ca

for each of the clusters installed.  It then creates a secret with one key per cluster `CLUSTER_CREDENTIALS_{1,2}`.

When the health-portal app is deployed - the secret is mounted as environment variables.

### CODE REVIEW

**Review by andrewh1978 — APPROVED** [review]

File: templates/health-portal-dr.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+scripts: ["install-awscli", "install-px", "licenses", "health-portal-credentials"]
+cluster:
+  - id: 1
+    scripts: ["health-portal-backups"]
+  - id: 2
+    scripts: []
+post_script: "health-portal-app"
+env:
+  px_suffix: "mon=true"
+  scenarios: "multiwriter,diskfull,backups,azoffline"
```
Comment (wallnerryan) [inline templates/health-portal-dr.yml:12]: Can we add a comment in this yaml that points to documentation at https://github.com/binocarlos/px-health-portal ?

File: templates/health-portal.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+description: Install and run Portworx on each cluster
+scripts: ["install-awscli", "install-px", "health-portal-credentials", "health-portal-backups"]
+post_script: "health-portal-app"
+env:
+  px_suffix: "mon=true"
+  scenarios: "multiwriter,diskfull,backups"
```
Comment (wallnerryan) [inline templates/health-portal.yml:6]: Can we add a comment in this yaml that points to documentation at https://github.com/binocarlos/px-health-portal ?

File: templates/health-portal-dr.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+description: Deploys 2 clusters with Portworx, sets up and configures a cluster pairing, configures an async DR schedule with a loadbalancer in front of the setup.
+clusters: 2
+scripts: ["install-awscli", "install-px", "licenses", "health-portal-credentials"]
+cluster:
+  - id: 1
+    scripts: ["health-portal-backups"]
+  - id: 2
+    scripts: []
+post_script: "health-portal-app"
+env:
```
Comment (wallnerryan) [inline templates/health-portal-dr.yml:10]: The readme defines username and password for UI. https://github.com/binocarlos/px-health-portal#readme Can these also be placed here? Also can we update to something healthportal specific like healthportal_user / healthportal_pass or something?

File: templates/health-portal.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+description: Install and run Portworx on each cluster
+scripts: ["install-awscli", "install-px", "health-portal-credentials", "health-portal-backups"]
+post_script: "health-portal-app"
+env:
```
Comment (wallnerryan) [inline templates/health-portal.yml:4]: The readme defines username and password for UI. https://github.com/binocarlos/px-health-portal#readme Can these also be placed here? Also can we update to something healthportal specific like healthportal_user / healthportal_pass or something?

File: templates/health-portal-dr.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+description: Deploys 2 clusters with Portworx, sets up and configures a cluster pairing, configures an async DR schedule with a loadbalancer in front of the setup.
```
Comment (wallnerryan) [inline templates/health-portal-dr.yml:1]: should description warn that this only works on AWS?

File: templates/health-portal.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+description: Install and run Portworx on each cluster
```
Comment (wallnerryan) [inline templates/health-portal.yml:1]: This seems to need to be updated for healthportal and should description warn that this only works on AWS?

**Pull request merged**